src/app/util/process-cluster-message.cpp

```diff
@@ -98,12 +96,6 @@ bool emAfProcessClusterSpecificCommand(EmberAfClusterCommand * cmd)
     }
 #endif
 
-    // Pass the command to the generated command parser for processing
-    status = emberAfClusterSpecificCommandParse(cmd);
-    if (status != EMBER_ZCL_STATUS_SUCCESS)
-    {
-        emberAfSendDefaultResponse(cmd, status);
-    }
-
-    return true;
+    // All cluster messages (commands) will go through the IM now, return false to indicate an error.
+    return false;
```

Comment:
Just curious. What about `global` commands, for example reading an attribute ? Is it going to still work using the old ZCL codepaths ?

Reply:
The global commands will still use the old codepaths.